X9.146/gen_falcon1_dual_keysig_cert.c

```diff
@@ -0,0 +1,379 @@
+/* custom_ext.c
```

Comment:
Note that since there is only 1 level, it does not need to be set on the commandline.  No need for a command line parameter that can only take one value.